#summary related records proposal

<table width="98%" border="0" cellpadding="10"><tr>

<td valign="top" width="*">

= Summary of the related records proposal =

== Introduction ==
This document is a draft proposal for handling relationships between data records. The intent of this proposal is to ensure the data standards and best-practice support communicating these relationships if the source data capture them.

*At 29/11/2011 this document is in draft form*; it supersedes (and subsumes) the earlier [IdentifyingRelatedRecords] document, also on this wiki.

== Relevance ==
This document applies to some of the entities represented in biological collection data (_i.e._ collection items). It focuses on the associations amongst specimen (or sighting) records, along with duplicates, other evidential data, or rich content (_i.e._ multimedia). 

It addresses two commonly-used specifications for biodiversity informatics data transfer: DarwinCore (2.x) and HISPID (5).

See _About the data_ and _Appendix: Relevance to different biodiversity domains_ for more information.

== Requirements ==
Note: _the following are prioritised using [http://en.wikipedia.org/wiki/MoSCoW_Method MoSCoW - http://en.wikipedia.org/wiki/MoSCoW_Method]_

=== General (functional) behaviour required of the collection management system ===
  # Each record that is to be shared must have a consistent and predictable unique identifier. The record will most likely represent a collection item, but may also represent other related entities, such as the collector, a location or a site visit;
  # This identifier is to be set by the data custodian, in accordance with a pattern agreed to by the relevant institutional group (i.e. informatics peak body);
  # If the record represents data that are derived from other data (see _About the data : Scenarios giving rise to relationships_) then this relationship should be stored in the collection management system, and communicated within the record using the appropriate method (see _About the data : Common patterns in the scenarios_);
  # Depending on the data-standard being used to share information, these identifiers will be stored in consistent and agreed ways (i.e. field names) so these relationships can be discovered programmatically;


=== Aggregator-specific behaviour ===
  # Due to the high likelihood of inconsistency, the data aggregator must not build a single composite record from related records that are determined to be duplicates or constituent parts, without highlighting the inconsistencies and clearly identifying the provenance of such a record;
  # The aggregator should use methods (see _About the data : Patterns of relationships_) to discover the existence of related records and clearly offer links to, or notification of, these records;
  # The aggregator could provide a (web?) service that:
    * responds to resource identifiers with a method for accessing the resource itself (freely or securely),
    * responds to the sub-parts of an identifier with best-guess identifier(s) based on current (and historical) patterns used by the relevant institutional group,
    * and potentially, offer the identifiers of related records as part of the response.

</td>

<td valign="top" align="right" width="33%">
<p> </p><p> </p>
<p><table border="1" cellspacing="0" cellpadding="4"><tr><td>
*Contents:*
<wiki:toc max_depth="3" />
</td></tr></table></p>
</td>

</tr></table>

----

= About the data =

== Core (primary) and supplementary (duplicated, derived, secondary) data ==

It seems highly unlikely that any attempt to apply labels to organisations' activities will not bring to light cases that contradict the reasoning used to assign these labels. Therefore, _core_ or _supplementary_ can be considered arbitrary labels used to distinguish the roles data play in relationships.

In the scope of this logic, *core collection data* is generally the data that find themselves at the source of other data (causally). They are distinguishable to *supplementary collection data*, which are usually derived in some way; _e.g._
  * a specimen (core) which is photographed, the photo being stored in a separate system (supplement);
  * a sound-recording (core) is taken in the field, within which sightings are evident (supplement);
  * a specimen (core) is scanned by a service provider with CT equipment (supplement);

The organisational boundary most likely plays a role in the distinction as well; _e.g._
  * the service provider keeps a database of all ct-scans; the specimen's ct-scan (core) is converted to a lower-fidelity .avi file and offerred on their website (supplement);
  * a herbarium manages a photo of a type specimen, held in another institution, within their own specimen management system (core);
  * _... it goes on!_

Suffice it to say, each type of data are likely handled in separate (sub-)systems with separate processes for managing them - this point forces us to reconsider some cases; _e.g._
  * a specimen (core) which then has its skin preserved and treated as a separate, but related, specimen (core);
 

== Scenarios giving rise to relationships ==

The following situations have been encountered to date; they are referred to later when itemising the possible solutions within each specification.

 # Specimen record holds metadata on multiple preparations
 # Specimen record is a single preparation
 # One or more preparations relate to one or more specimens through a cross-reference table (_note: this situation has not been encountered - if this is your data, give us a yell! thanks..._)
 # Any of the above three, but 'specimen' is already a constituent part (_piece, relationship, derivation_) of another
 # A 'collection item' being a generalisation of the more specific (specimen) in any of the above scenarios
 # Any collection item, from which one or more observations (sightings, occurrences) are evident - these observations may not necessarily have a determination (taxon)
 # Any collection item representing another in a many to one relationship; eg:
   * a photo of a drawer full of specimens
   * a photo of a flock of birds
   * a digitisation of a sound-reel, having many recordings, each recording a discrete site visit, each recording capturing one or more taxa
 # DNA and one (or more) sequence(s) of (barcodes), each sequence matching one or more taxa (next-gen sequencing - http://www.google.com.au/search?q=nextgen+sequencing)
 # vouchering of specimens amongst herbaria
 # duplication of specimens within herbaria collection management systems (digital representations of the physical, sharing, donation
 # collaboration between institutions in the field; specimen(s) gathered from a common observation (biological individual, sighting, occurrence) made on a joint expedition
 # deriving sightings from sightings (?)
 # ... 


== Common patterns in the scenarios ==

This section attempts to identify instances of common ground shared amongst the scenarios, i.e. a 'lowest common denominator', with the goal of relating each pattern to a prescribed usage within a particular standard. 

(For some thoughts on how these patterns are identified, see _Appendix: Searching for common patterns in the scenarios_)

_can i embed a google docs spreadsheet in this wiki page? that would be good!_ [http://goo.gl/PsS0y]

<wiki:gadget url="http://hosting.gmodules.com/ig/gadgets/file/117808631063490062819/url.xml" up_Url="https://docs.google.com/spreadsheet/pub?hl=en_US&hl=en_US&key=0AiNWJFdh4pHZdDY2cEgxdHZ5YlBoTUx4MGVfVTgyTGc&single=true&gid=0&output=html" up_Title="matrix" up_height=510 width=768 up_refresh=6000 />

----

= Standards-specific usage patterns =

== Introduction ==

It is the intent of this proposal to define consistent methods for communicating relationships between records; in all cases, the following design-constraints are considered:
  # be flexible but solve the problem, and accept that there are multiple solutions - let the data provider choose the best fit;
  # use the existing standard terms where ever possible, unless the use of these terms in this manner would cause a conflict with the general understanding and/or current practice;
  # add terms only when the following conditions are met:
    # the term name is defined which at least hints at the purpose (_yes, this is purely subjective - but if a number of people agree that the naming makes sense, then that's the best that can be hoped for at this stage_)
    # the term's usage is unambiguously defined in a short description

For true 'backwards compatibility', the newer (extended) version of records must conjur the same behaviour in consuming applications as the earlier version would have.

*One assumption must be made in this regard:* that the data consumer ignores unrecognised fields instead of calling a halt to the load process, or, at worst case, drops this particular record as 'an error'. Ideally, unrecognised fields would be ignored.

In most cases this is possible, however, in some instances where relationships between duplicate records are not clearly defined then multiple 'dots on the map' will appear. It is the hope that in the future some mechanism exists to mine for, seek validation and then remember these relationships - see _Appendix: Relationship mining sub-system (The Relator)_ for more information.


== Changes to the DwC specification ==

Each of these terms should be added to the FCIG dwc extension; propose for ratification+inclusion in DwC
  # add _principalRecord_
  # add _parentRecord_
  # add _principalRelationshipType_
  # add _parentRelationshipType_

== Changes to the HISPID specification ==
Each of these terms should be added to the HISCOM ABCD extension (HISPID5):
  # add _principalUnitID_ to _MultiMediaObject_ and _SiteImage_
  # add _parentUnitID_ to _MultiMediaObject_ and _SiteImage_


== DarwinCore usage patterns ==

=== The principal record ===
The identifier assigned to the physical item that was _"accessioned into a collection from the field"_ (eg. single vouchered specimen, still image, movie or sound recording, field notes, core sample, phial of multiple organisms, etc.). 

The principal record may be associated with many occurrence records, each with a unique occurrenceId (eg. an image or sound recording containing determinations for many species) as well as derivative records such as gene sequences representing a part of the physical item (eg. leg of spider), or unique records representing different layers of a core sample, etc.

Examples of cases: 
  * downstream preparations/processing of specimens, such as a dna sequence, or a skin preservation, etc. would use this term to 'point back to the originating specimen' 
  * a record of a taxon in multimedia (hosting many taxa) would point to the accessioned collection item, e.g. a sound recording of one or more species, or a photo of an apparent symbiotic relationship (although in this case, "associated taxa" or "associated occurrences" might be more appropriate if the nature of the relationship can be more specifically defined)

=== The parent record ===
if the principal record is more than one step removed from this one, you can use this term to communicate the direct parent-child relationship between these two items (_see principal record for more detailed information_)

=== Preparations versus basis of record ===
  * The concepts 'basis of record' (DwC: basisOfRecord + dcterms:type; HISPID: xxx) and preparations  (DwC: preparations; HISPID: xxx) are metadata.
  * source data – does each record represent a preparation? or are all preparations for a principal collection item aggregated into the one record?

== Potential HISPID5 usage patterns ==
[http://goo.gl/50KhE http://goo.gl/50KhE _Specimen / Image Relationships with HISPID 5_]

[https://docs.google.com/document/d/1bJYvz0AXT-O1iiJUs-ongmngTAhDiMfrJD2vu_5lmmM/edit?hl=en_US https://docs.google.com/document/d/1bJYvz0AXT-O1iiJUs-ongmngTAhDiMfrJD2vu_5lmmM/edit?hl=en_US _Specimen / Image Relationships with HISPID 5_]

This section looks at the problem from the hispid perspective; this is what i want to do in hispid - what cases does it apply to? Here is a crude glossary:

  * images - read ‘multimedia’; the assumption is that these are at least indexed in digital form somewhere (with appropriate metadata to find the object, if the image itself has not been digitised);
  * storage mechanism - the database (/filestore/card-file...) that holds the data;
  * ‘separate storage mechanisms’ imply a challenge in integrating the data; some types of the impediments encountered so far are a lack of technical capability, a physical separation & a temporal separation;
  * Unit - HISPID
  * HISPID SiteImage, SI - DataSets/DataSet/Units/Unit/Gathering/SiteImages/SiteImage http://wiki.tdwg.org/twiki/bin/view/ABCD/AbcdConcept1138
  * MultiMediaObject, MMO - DataSets/DataSet/Units/Unit/MultiMediaObjects/MultiMediaObject http://wiki.tdwg.org/twiki/bin/view/ABCD/AbcdConcept0772
  * SpecimenUnit, SU - DataSets/DataSet/Units/Unit/SpecimenUnit http://wiki.tdwg.org/twiki/bin/view/ABCD/AbcdConcept0404

=== SpecimenUnit record with nested SiteImage record(s) ===

straightforward way of handling scenario XXX; the storage mechanisms are not separated in any way; a complete SpecimenUnit record with nested SiteImage record(s) can be built at the time of export;

=== SpecimenUnit record with nested MultiMediaObject record(s) ===

straightforward way of handling scenario XXX; the storage mechanisms are not separated in any way; a complete SpecimenUnit record with nested MMO records can be built at the time of export;

=== MultiMediaObject/SiteImage record stands alone, from which sighting (specimen?) records are interpolated ===

(this may not be a valid case in the herbaria domain) - in one of the faunal collections, the collection specimen is the sound recording of a habitat; from this, taxa are identified as existing in the fore/background - an occurrence (observation, sighting) is reported for each 'foreground taxon', while background taxa may have their own records, or may be referred to in metadata of the foreground record(s);

HISPID would allow for this but each inferred SpecimenUnit record must have a nested MMO/SI record - in turn, each MMO/SI record would be a redundant copy of the other(s), with all the MMO/SI records sharing the same identifier.

=== An image collection item is treated like more traditional specimen(s); this image may be related to other more traditional specimen(s) ===

In this case, the image is the SpecimenUnit; it may be a duplicate, or derivation, of other SpecimenUnit(s); it has nested MultiMediaObject(s) to point to the image where it resides (usage pattern 1).

    The image may be managed in a digital asset management system (DAMS) or it might be simply a file sent on the email from another institution.
    It may be desirable as the aggregator to keep both source records (the whole may be equal to or greater than the sum of the parts); however, in this case, the relationship between both records must be maintained.


=== SpecimenUnit record and MultiMediaObject / SiteImage record stand alone but share a 1:1 relationship ===

‘separate storage mechanisms’ imply some challenge in integrating the data; up until this point, it is assumed that the data provider will be able to build an aggregate SpecimenUnit record at the time of export, when they are communicating imagery associated with the specimen;

some types of the impediments to this ‘just-in-time aggregation’ that have been encountered so far are a lack of technical capability, a physical separation & a temporal separation;

MultiMediaObject or SiteImage records are nested under Unit (spec suggests that MMO/SI is part of unit); however, MMO/SI has no way of communicating that it belongs to Unit - without this mechanism, each of these types of records could not exist in separate files, yet still maintain their relationship;

this usage pattern doesn’t appear to be supported by hispid as it stands; we would need to add a term to MMO/SI to point to its related Unit - so it is anticipated that the main users of such a mechanism would have a need to communicate SpecimenUnit records completely separately  to their image records, yet have a desire that these be related.

----

= Appendices =

== Appendix: Relevance to different biodiversity domains ==

This document is relevant to a number of different groups sharing biodiversity information. The boundaries may be :
  * historical (or traditional), 
  * along taxonomic lines (or other taxa classification methodology),
  * focussed on activities of the organisations within the group (i.e. areas of expertise),
  * any or all of the above!

|| *Domain* || *Description of focus* || *Data sharing standard* || 
|| FCIG || Faunal collections || Darwincore csv (simple-dwc) with non-standard domain extensions to be proposed to TDWG, and maintained by FCIG || 
|| HISCOM || Herbaria || HISPID5: hispid-xml and 'hispid-light' || 
|| Microbial collections || Darincore archive (dwc-a) with domain extensions based on ITF2 standard<br>hispid  || 
|| Barcoding & sequencing || Darwincore csv (simple-dwc) || with non-standard extensions || 
|| Living collections || Darwincore csv (simple-dwc) || with domain extensions based on ITF2 standard || 


== Appendix: URNs as record identifiers ==

Uniform resource names (URNs) are textual data intended for use as identifiers of other data or services (resources), and display the following attributes:
  * They must follow a certain pattern and contain information identifying the custodian of the resource (the namespace), as well as the resource itself (the name).
  * Patterns that URNs follow must be predictable, so data users can derive them from information they may already have (if they don't have the URN itself).
  * URNs must remain consistent (i.e. always point to the same thing).
  * URNs are defined by the data custodian, and may follow a pattern or rules specified by the custodian or a broader affiliation of custodians (e.g. a biodiversity domain).
  * URNs need not be 'doled out by a central repository' although some namespaces or broader groupings do this as part of a standard.
  * Given only an identifier, a data user should be able to find more information from someone or something about the resource itself (if not the resource).

=== LSIDs ===

For the purposes of this document, LSIDs are synonymous with URNs??
==== Structure of, pattern rules ====
==== Basis of record contained within LSID ====
==== Historical mappings to earlier patterns ====

=== LSID responder (URN resolution) ===
For the purposes of this document, LSIDs are synonymous with URNs??
==== Intended behaviour ====
==== Output ====
==== Multiple matches to a LSID ====


== Appendix: Searching for common patterns in the scenarios ==

The section _About the data : Common patterns in the scenarios_ attempts to identify instances of common ground shared amongst the scenarios, _i.e._ a 'lowest common denominator', with the goal of relating each pattern to a prescribed usage within a particular standard.

A standing assumption: relationships between records are captured in the data management processes, and are evident in the source data.

=== A little about patterns ===
A pattern can be thought of as a generic method for solving a particular problem. It doesn't delve into specifics per se, rather, defines the requirements for a solution.

It is then up to the implementation to ensure the requirements are met within its own context (_more info: [http://en.wikipedia.org/wiki/Design_pattern_%28computer_science%29]_).

It could be said that the scenarios themselves are actually instances of a particular solution to a problem, each within a given context and/or domain. A solution exists within its source-system's boundaries, generating and/or capturing data relevant to its activities - this forms part of your operational data store.

=== Why bother identifying these patterns? ===
When it comes time to map these data to a standard form, we must ensure the relationship information continues to be discoverable in an explicit manner: at the business end, we need a way to handle each of the 'common denominators' in each of the data standards we are referring to in this document.

In addition to this, it is with some certainty that scenarios will arise that are not in the above list; however, a method for solving that problem will have already been defined.

Some of these methods will be applicable to the specifications as they stand, and others will require a currently non-standard solution. This is clearly defined in _Standards-specific usage patterns_.

=== More on relationships ===
In database parlance, the _cardinality_ of a particular relationship is defined in a handful of ways:  one-to-one, one-to-many (/many-to-one) and many-to-many (_see [http://en.wikipedia.org/wiki/Cardinality_%28data_modeling%29 http://en.wikipedia.org/wiki/Cardinality_%28data_modeling%29]_).

The right choice depends on the process and data being captured; but the method employed to communicate these will likely differ even within the same circumstance. 


== Appendix: Notes ==

{{{
_OLD: this is the table of contents for the older version of this doco
Data mobilisation: related records proposal
1	SUMMARY	1
1.1	SPECIFIC RELEVANCE TO DIFFERENT BIODIVERSITY DOMAINS	2
2	URNS AS RECORD IDENTIFIERS	3
2.1	LSIDS	3
2.1.1	Structure of, pattern rules	3
2.1.2	Basis of record contained within LSID	3
2.1.3	Historical mappings to earlier patterns	3
2.2	LSID RESPONDER (URN RESOLUTION)	3
2.2.1	Intended behaviour	3
2.2.2	Output	3
2.2.3	Multiple matches to a LSID	3
3	RELATIONSHIPS BETWEEN RECORDS	4
3.1	CORE (PRIMARY) AND DERIVED (SECONDARY) COLLECTION DATA	4
3.1.1	The principal record	4
3.1.2	The parent record	4
3.2	PREPARATIONS VERSUS BASIS OF RECORD	4
3.3	USAGE SCENARIOS AND EXAMPLE DATA	4
4	RELATIONSHIP MINING SUB-SYSTEM (THE RELATOR)	8
4.1	VISION AND DESCRIPTION	8
4.2	EXISTING FUNCTIONALITY	8
}}}

|| 
{{{
3.3	Usage scenarios and example data
Somewhere in the following morass lies a few use-cases!

Hi all,
This is a feast that is forever on the move so any attempt to tie it down so we can all partake would be much appreciated. 
Just some points id like to make:
1.       “having a framework in place that we can, at least aim at” is a good goal – I totally agree, however the trouble is that this is exactly what we have been trying to do for the past 5 years and things keep moving and shifting on us giving the impression that the topic is always under discussion. 
2.       If we were to have waited until we got the unique identifier fool proof then we wouldn’t be sharing any data at this point – we need to keep perspective and do our best 
3.       Even under these circumstances specimen records should have some metadata that never change – ie catalogue number and collection and institution identifiers. Until we have lsid resolvers working across a good swag of records then really as long as the occurrence-id contains those three elements then a record is uniquely identifiable at least to a particular organism - tho there may be multiple records for a catalogue number it should still relate to the one organism.
4.       Is it such a bad thing that a search on an lsid brings back multiple records? – as long as they all relate to the one organism? Ie id be pretty happy to get back 5 records for the one lsid as long as they all are different types of media/preparations/basis or record of the one organism eg a sound file, a specimen, a dna sequence – rather than having 5 separate lsids for all of these. Even different versions would be ok
5.       My point from 4 is that many records and institutions may not be able to achieve the best practice lsid (just as Paul A suggests) but that hardly renders it useless – rather as long as we have a strong flexible basis – ie like a core or mandatory lsid and then optional extras – then we can still make them useful.
6.       By all means lets aim for a good strong flexible framework – but also lets not lose sight of the fact that aiming for a system that can cope with every possible permutation can be counter productive. We should be aiming to deliver the maximum amount of data in a way that is doable with limited resources. Delivering the final 5% is sometimes not worth the effort. 
Cheers
Paul

PS looking forward to the outcome of Thursdays meeting


From: Paul Avern [mailto:paul.avern@qm.qld.gov.au] 
Sent: Monday, 29 August 2011 1:40 PM
To: Garry.Jolley-Rogers@csiro.au; Bryn.Kingsford@csiro.au; Margaret.Cawsey@csiro.au; David.Martin@csiro.au; Peter.Brenton@csiro.au; Paul Flemons; Ewallis@museum.vic.gov.au; piers@gaiaresources.com.au; Peter.Doherty@csiro.au; Miles.Nicholls@csiro.au; Alex.Drew@csiro.au; Bryan.Kalms@csiro.au
Subject: RE: occurrenceID & also basisOfRecord > + meta data and media files.

Hi All
 
I agree with Garry on this point. I vote for having a framework in place that we can, at least, aim at. Whether we ever achieve this 'ideal' as individual institutions is up to us. However, we can't go on pumping out batches of specimen records with different occurrenceIDs each time we export, simply because the topic is still 'under discussion'.
 
Same goes for images.
 
Cheers
Paul

________________________________________
From: Garry.Jolley-Rogers@csiro.au [mailto:Garry.Jolley-Rogers@csiro.au] 
Sent: Monday, 29 August 2011 12:48 PM
To: Bryn.Kingsford@csiro.au; Margaret.Cawsey@csiro.au; David.Martin@csiro.au; Peter.Brenton@csiro.au; paul.flemons@austmus.gov.au; Paul Avern; Ewallis@museum.vic.gov.au; piers@gaiaresources.com.au; Peter.Doherty@csiro.au; Miles.Nicholls@csiro.au; Alex.Drew@csiro.au; Bryan.Kalms@csiro.au
Subject: RE: occurrenceID & also basisOfRecord > + meta data and media files.
Bryn,
Good points.  As you suggest, each difficulty can be dealt with in turn and with engagement.  
Seems to me that, whilst communities don’t want to be dictated too – they also are seeking a framework to respond too / rebel against.  You are well  placed to identify patterns. More coffee? My shout this time.

G

From: Kingsford, Bryn (CES, Black Mountain) 
Sent: Monday, 29 August 2011 12:30 PM
To: Jolley-Rogers, Garry (PI, Black Mountain); Cawsey, Margaret (CES, Crace); Martin, Dave (CES, Black Mountain); Brenton, Peter (CES, Black Mountain); Paul Flemons; paul.avern@qm.qld.gov.au Avern; Ely Wallis; Piers Higgs; Doherty, Peter (CES, Black Mountain); Nicholls, Miles (CES, Black Mountain); Drew, Alex (CES, Crace); Kalms, Bryan (CES, Black Mountain)
Subject: RE: occurrenceID & also basisOfRecord > + meta data and media files.

g'day all
 
some thoughts on possible traps with the earlier proposal
 
 - where multimedia are derived from specimen records (or vice versa) you still need a way to associate the file with the specimen record; this is the job of proposed new dwc term (currently named) 'principal collection item'
 
 - dwc spec 'basis of record' suggests best practice to use a controlled vocab, but this vocab doesn't adequately convey microbiology (amrin), digital tracks derived from an open reel (anwc) and probably more (sequencing??) - my gut feeling here is to encourage the communities to agree on controlled vocabs, without dictating - this is before we even get to cases of misuse; i.e. assume nothing in here either
 
 - on the topic of 'basis of record', the lsid 'occurrence id' alone is not enough to uniquely identify a dwc record. if this is your key then 
be prepared to receive multiple answers under the following scenarios:
1. multiple preparations of a specimen within the institute
2. derivations of a specimen, ie, digital/dna/sequence/...
3. vouchering of specimen(s)
4. newer versions of the same record
 
 - dwc has no specific concept of a 'version number' that i can think of off the top of my head; 'dcterms:modified' (date record was last modified) is the closest thing to this (also a fundamental part of basis)
 
 - where multimedia represent anything but specimens/occurrences, they might break the registration system in an institute; good to recommend best practise, bad to expect it. (eg: sound representing taxon, photo of a habitat, ...)
 
---
 
some scenarios for you to test assumptions with (note, these might stretch beyond the scope of the lsid resolver, but i believe they aren't beyond the scope of identifiers in dwc - happy to discuss further):
 
1. find the newest specimen record associated with a lsid
 
2. find any record associated with a 'catalogue/registration number/accession id/...'
 
3. find all image(s) that relate to a specimen lsid
3b. find all specimen records that relate to an image lsid
 
4. find dna that came from a specimen, identified by a lsid:
a) within the same institution
b) within a different institution
 
5. find evidence of vouchering of specimens in other institutes from a given lsid
 
6. remove unnecessary 'dots on the map' in the above
 
r's bryn
 
 
 

________________________________________
From: Jolley-Rogers, Garry (PI, Black Mountain) 
Sent: Friday, 26 August 2011 15:35
To: Cawsey, Margaret (CES, Crace); Kingsford, Bryn (CES, Black Mountain); Martin, Dave (CES, Black Mountain); Brenton, Peter (CES, Black Mountain); Paul Flemons; paul.avern@qm.qld.gov.au Avern; Ely Wallis; Piers Higgs; Doherty, Peter (CES, Black Mountain); Nicholls, Miles (CES, Black Mountain); Drew, Alex (CES, Crace); Kalms, Bryan (CES, Black Mountain)
Subject: RE: occurrenceID & also basisOfRecord > + meta data and media files.
Hi, 

Interesting how tech notes are often so much shorter than plain English “translations” – see Dave’s notes below.  

Below you will find notes from a meeting over coffee between Dave and I pertaining to
•         occurrence record identifiers
•         identifiers and metadata  for digital media 

We would appreciate your comments. Is this practical? nb mistakes are my fault.

Key points
1.       digital media objects can be treated in much the same way as occurrence records for specimens
a.       an appropriate record describing the media file (determined by the provider)
                                                               i.      with pertinent (DwC ?) information served via OZCAM
                                                             ii.      with an appropriate Basis of Record 
(do the terms for basis of record need to be agreed and enumerated?)
b.      a  unique id (in the collection); 
either from a DAMS or 
treating the file as an accessioned part of the collection
c.       in as much as it is possible and practical, lsid’s and other metadata be embedded in the media.  

This seems to cover all the bases. 
Best practice would then be to embed the identifier and  metadata in the digital object. 
Good practice would be to accession digital media objects or use a DAMS or similar.
2.        a strategy for handling the resolution  of lsid’s  and return of metadata   - (see below).
3.      variations of lsid forms  to be handled by the resolver.
e.g.  (punctuation with either dots or colons)
urn:lsid:ozcam.taxonomy.org.au:[Institution Code]:[Collection code]:[Basis of Record]:[Catalog Number]:[Version]
urn:lsid:ozcam.taxonomy.org.au:[Institution Code].[Collection code].[Basis of Record].[Catalog Number];[Version]
}}} 
||